Added minimal permissions to Entra enterpriseapp commands  Closes (#6927)

santhosh-7777 opened the pull request:
This pull request updates the documentation for the following m365 entra enterpriseapp commands:

add → Added Delegated and Application permissions: Application.ReadWrite.All

get → Added Delegated and Application permissions: Application.Read.All

list → Added Delegated and Application permissions: Application.Read.All

remove → Added Delegated and Application permissions: Application.ReadWrite.All

The Permissions section is now included between Examples and Response in each command’s documentation.

No code changes were made.  this is purely a documentation update.

Closes #6927

Adam-it commented:
Thank you for opening the PR 👏
We will review it ASAP

MartinM85 reviewed (changes requested):
Nice work @santhosh-7777 🙂. 
I have a couple of small comments, nothing major.
Could you please look at it?

santhosh-7777 commented:
Hi @MartinM85,
I’ve updated the documentation as per your comments — corrected the resource names, permissions tables, and removed extra blank lines.
Please have a look when you get a chance.

Thanks!
Santhosh
